### PR TITLE
[SHLWAPI] lang/*.rc : Enforce aa-BB filenames

### PR DESCRIPTION
**Pull request by SergeGautherie**

### CODE REVIEW

**Comment by binarymaster** [comment]
Is this synced with 🍷 ?

**Comment by SergeGautherie** [comment]
> Is this synced with 🍷 ?

Not these parts:
https://source.winehq.org/git/wine.git/blob/HEAD:/dlls/shlwapi/shlwapi.rc

**Comment by HBelusca** [comment]
These (old) file names are those that come out when you generate the corresponding language resource files out of the *.po for shlwapi. The same thing should also happen for all the other wine-synced dlls.

**Comment by SergeGautherie** [comment]
> all the other wine-synced dlls

Yes, first search returns about 20+ similar cases.

File: dll/win32/shlwapi/shlwapi.rc [hunk 1]
```diff
@@ -30,77 +30,77 @@
...
 #ifdef LANGUAGE_RU_RU
-    #include "lang/shlwapi_Ru.rc"
-#endif
-#ifdef LANGUAGE_SK_SK
-    #include "lang/shlwapi_Sk.rc"
+    #include "lang/ru-RU.rc"
 #endif
 #ifdef LANGUAGE_SL_SI
-    #include "lang/shlwapi_Si.rc"
+    #include "lang/si-SI.rc"
```
Comment (ThFabba) [inline dll/win32/shlwapi/shlwapi.rc:87]: The file should be called `sl-SI` (es el, ES EYE)

Comment (SergeGautherie) [reply dll/win32/shlwapi/shlwapi.rc:87]: Good catch :-) Fixed.

**Comment by SergeGautherie** [comment]
> I'm not against this but I don't think it's worth spending time on.

Then, I filed [CORE-17110](https://jira.reactos.org/browse/CORE-17110).

**Comment by ThFabba** [comment]
> > I'm not against this but I don't think it's worth spending time on.
> 
> Then, I filed [CORE-17110](https://jira.reactos.org/browse/CORE-17110).

I changed my mind. I am now actively against making more of these changes.

**Pull request merged**